### ISSUE TITLE
Widget property editing modeless dialog in IntelliJ

### ISSUE THREAD

**Issue by jacob314** [flutter/flutter-intellij#4026]
The dialog should be productively keyboard navigable so that it is natural as a replacement for the quick-fix dialog when a property or body of a widget constructor is selected.
